app/src/test/java/comp3350/mealbuddy/tests/objects/FoodTest.java

```diff
@@ -0,0 +1,134 @@
+package comp3350.mealbuddy.tests.objects;
+import org.junit.Assert;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import comp3350.mealbuddy.objects.consumables.Edible;
+import comp3350.mealbuddy.objects.consumables.Edible.Macros;
+import comp3350.mealbuddy.objects.consumables.Edible.Micros;
+import comp3350.mealbuddy.objects.consumables.Food;
+
+
+public class FoodTest {
+
+    public static final int TEST_VALUE = 20;
```

Comment:
This is not a good name.  I have no idea what this means.  This might as well be a magic number